cdk/lib/dynamo-policy.ts

```diff
@@ -0,0 +1,20 @@
+import type {GuStack} from "@guardian/cdk/lib/constructs/core";
+import {GuAllowPolicy} from "@guardian/cdk/lib/constructs/iam";
+import type {Table} from "aws-cdk-lib/aws-dynamodb";
+
+const readActions = ["BatchGetItem", "GetItem", "Scan", "Query", "GetRecords"];
+const writeActions = ["BatchWriteItem", "PutItem", "DeleteItem", "UpdateItem"];
```

Comment:
these permissions are taken from the gu-cdk construct here:
https://github.com/guardian/cdk/blob/main/src/constructs/iam/policies/dynamodb.ts#L31